source/linux/networking/network_config.rst

```diff
@@ -0,0 +1,62 @@
+Chaning network config
+======================
+
+ifconfig
+^^^^^^^^
+    Description:
+        This command is used to show the current network configuration.
+        It can also be used to change the network configuration.
+
+    Usage:
+        ifconfig [interface] [address] [netmask] [broadcast] [metric] [mtu] [hwaddr] [up|down]
+        
+    Example:
+        .. code-block:: bash
+
+            host$ ifconfig
+        
+        shows the current network configuration.
+
+
+        .. code-block:: bash
+
+            host$ ifconfig <adapter> <address> netmask <netmask>
+
+            Changes the address of a specific network adapter until the next restart
+
+
+        .. code-block:: bash
+
+            host$ ifconfig <adapter> 0.0.0.0 0.0.0.0 && dhclient
+
+        Changes the address of a specific network adapter to DHCP
+
+
+ping
+^^^^
+    Description:
+        This command is used to send ICMP echo requests to a host.
+        You can use it to check if a connection is working.
+
+    Usage:
+        ping [options] [host]
+
+    Example:
+        .. code-block:: bash
+
+            host$ ping <address>
+
+ip addr
```

Comment:
Some more information about ip would be nice, since it will replace ifconfig in the most distros